src/index.html

```diff
@@ -39,9 +39,9 @@ <h2>Novos hacks toda semana</h2>
         {% include next-event-table.html %}
     </div>
     <div class="home--section home--color-scheme-5">
-            <div class="home--content home--right-text">
-                <h2>Quartas-feiras,<br> às 13h15</h2>
-                <p>No Auditório do CCSL, no IME-USP.</p>
+            <div class="home--content home--color-scheme-5">
```

Comment:
doesn't the upper div already contain this class?

Reply:
Yes, it does... I don't know why I thought it was necessary